### ISSUE TITLE
Add support for pdf and abstract urls

### ISSUE THREAD

**Issue by george-gca** [asreview/asreview#1450]
## Feature Request

**Is your feature request related to a problem? Please describe.**
It would be nice to have links to both [abstract](https://openaccess.thecvf.com/content/CVPR2023/html/Ci_GFPose_Learning_3D_Human_Pose_Prior_With_Gradient_Fields_CVPR_2023_paper.html) and [pdf](https://openaccess.thecvf.com/content/CVPR2023/papers/Ci_GFPose_Learning_3D_Human_Pose_Prior_With_Gradient_Fields_CVPR_2023_paper.pdf) as options in the UI.

**Describe the solution you'd like**
Maybe a `pdf_url` or `abstract_url` field in the [data format](https://asreview.readthedocs.io/en/latest/data_format.html) would be nice.

**Teachability, Documentation, Adoption, Migration Strategy**
This way, when viewing the paper abstract, one could go directly to the paper, or to the abstract url, where more info could be found, like supplementary material as in the example provided.

**Comment by Rensvandeschoot** [comment]
Is this information always available in the meta-data?

**Comment by FelixWdm** [comment]
dear @george-gca for what data sources would you expect these fields? They don't seem common, but if there's a specific source or field they are used, we'll look into it. Thanks!

**Comment by george-gca** [comment]
I don't think this is commonly found in the metadata, but if I am manually creating the data is something that I can add, what is my specific use case. If I had a way to link both pdf and abstract separately, would be a nice addition.

**Comment by Rensvandeschoot** [comment]
Would you be interested in drafting a PR for adding such functionality?

**Comment by george-gca** [comment]
Sure, I will give it a go.

**Comment by george-gca** [comment]
@Rensvandeschoot question: how do I setup local dev environment? I cloned the repo, made a few changes, and now I want to launch ASReview from local directory. How do I do that? Couldn't quite find it in the [docs](https://asreview.readthedocs.io/en/latest/overview_development.html).

**Comment by rohitgarud** [comment]
Hi @george-gca, run the following command from inside the local asreview directory:
```python
pip install -e .
```
or 
```python
pip install .
```
Pip will install the updated version in your dev env and then you can launch `asreview lab` from the env terminal

**Comment by Rensvandeschoot** [comment]
see the development documentation here: https://github.com/asreview/asreview/blob/master/DEVELOPMENT.md

**Comment by george-gca** [comment]
> Hi @george-gca, run the following command from inside the local asreview directory:
> 
> ```python
> pip install -e .
> ```
> 
> or
> 
> ```python
> pip install .
> ```
> 
> Pip will install the updated version in your dev env and then you can launch `asreview lab` from the env terminal

Just tried this with pipenv, but got some errors that I posted in #488
